deployment/logs-benchmark/readme.md

```diff
@@ -0,0 +1,74 @@
+# Benchmark for VictoriaLogs
+
+Benchmark is based on:
+
+- Logs from this repository - https://github.com/logpai/loghub
+- filebeat - https://www.elastic.co/beats/filebeat
+- elastic + kibana
+- [logs generator](./generator)
+
+## How it works
+
+docker-compose.yml contains:
+
+- 2 filebeat instances - one for elastic and one for VictoriaLogs.
+- elastic instance
+- kibana instance - port forwarded to `localhost:5601` to see UI
+- vmsingle - port forwarded to `localhost:8428` to see UI
+- [logs generator](./generator)
+- exporters for filebeat/system
+
+[Logs generator](./generator) generates logs based on logs located at `./source_logs/logs` and sends them to filebeat
+instances via syslog.
+Logs are generated by reading files line by line, adding randomized suffix to each line and sending them to filebeat via
+syslog.
+By default, generator will exit once all files are read. `docker-compose` will restart it and files will be read again
+generating new logs.
+
+Each filebeat than writes logs to elastic and VictoriaLogs via elasticsearch-compatible API.
+
+## How to run
+
+1. Download and unarchive logs by running:
```

Comment:
Please add the approxiamate size of uncompreesed logs and how to exclude each of them from downloads